src/lager_gelf_formatter.erl

```diff
@@ -17,35 +17,37 @@ format(Message,[{metadata, AdditionalMetaData} | Rest]) ->
     Content = lager_msg:message(Message),
     NewMessage = lager_msg:new(Content, Timestamp, Severity, Metadata, Destinations),
```

Comment:
Life is short. How about running a code beautifier before turning on gadgetci? There is no incentive to provide upstream fixes otherwise.